freebasics/templates/core/tags/latest_listing_homepage.html

```diff
@@ -4,7 +4,8 @@
 {% if articles %}
   {% for article in articles %}
   <div class="block-article fb-body fb-card">
-    <h6 class="block-heading fb-accent-1">{{article.title}}</h6>
+    <a href="{% pageurl article %}" class="no-underline"><h6 class="block-heading fb-accent-1">{{article.title}}</h6></a>
```

Comment:
if the a is in the h6 tag it changes the css. we have no underline here because an href generally has a line underneath it, the others already have no underline in their css